PubnativeLite/PubnativeLite/Ad Model/HyBidSkAdNetworkModel.h

```diff
@@ -0,0 +1,27 @@
+//
+//  HyBidSkAdNetworkModel.h
+//  HyBid
+//
+//  Created by Orkhan Alizada on 18.09.20.
+//  Copyright © 2020 Can Soykarafakili. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "HyBidBaseModel.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol SKAdNetworkDelegate<NSObject>
```

Comment:
This should be prefixed with HyBid as well. Since we might get duplicated symbols if another library in the same publisher app uses this name. We've had this issue before with classes we didn't even imagine could cause it.